core-bundle/src/Resources/contao/library/Contao/Input.php

```diff
@@ -39,23 +39,23 @@ class Input
 	/**
 	 * Parameters set via setGet() are stored by request
 	 *
-	 * @var \WeakMap<Request,array<string,array|string>>
+	 * @var \WeakMap<Request,array<string,array|string>>|null
 	 */
-	private static \WeakMap $setGet;
+	private static \WeakMap|null $setGet = null;
```

Comment:
I’d prefer making then not nullable, see https://github.com/leofeyer/contao/pull/4/files
WDYT?